client/signup/steps/p2-details/index.jsx

```diff
@@ -78,7 +78,7 @@ function P2Details( {
 				</div>
 				<div className="p2-details__description">
 					{ translate(
-						'P2 is powered by WordPress.com: log in with your account (or create a new one) ' +
+						'P2 is powered by WordPress.com: log in to your account (or create a new one) ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'P2 is powered by WordPress.com: log in with your account (or create a new one) to continue.' )` *ES Score: 8*